test/e2e-tests.js

```diff
@@ -384,6 +415,17 @@ describe('matrix-public-archive', () => {
         replyMessageElement.outerHTML,
         new RegExp(`.*${escapeStringRegexp(normalMessageEventId2)}.*`)
       );
+
+      const replyMissingRelationMessageElement = dom.document.querySelector(
+        `[data-event-id="${replyMissingRelationMessageEventId}"]`
+      );
+      // Make sure the reply text is there.
+      // We don't care about the message we're replying to because it's missing on purpose.
+      assert.match(
+        replyMissingRelationMessageElement.outerHTML,
+        new RegExp(`.*${escapeStringRegexp(replyMissingRelationMessageText)}.*`)
+      );
```

Comment:
This PR is mainly just about adding a test for this case since it failed before https://github.com/matrix-org/matrix-public-archive/pull/51